src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/compute/v2/AwsV2ComputePlugin.java

```diff
@@ -280,7 +254,7 @@ private String addPublicIpAddress(Instance instance, int index) {
 		return ipAddress;
 	}
 
-	private List<String> addPrivateIpAddresses(Instance instance, int index) {
+	private List<String> retrievePrivateIpAddresses(Instance instance, int index) {
```

Comment:
do we need this check for `isEmpty`?

Reply:
that's true, there is no need to check it